src/components/Card/Card.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+import { ExternalStyles, useCss } from '../../styles'
+import { CardVariant, createBaseStyles, createVariantStyles } from './CardVariants'
+
+export interface CardProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'style'> {
+  style?: ExternalStyles
+
+  /**
+   * Defines the visual style of the Card
+   * @default 'outline'
+   */
+  variant?: CardVariant
+
+  /**
+   * Specify whether the Card should have a error appearance
+   */
+  error?: React.ReactNode
+
+  /**
+   * Specify whether the Card should have a disabled appearance
+   */
+  disabled?: boolean
+}
+
+export function Card(props: CardProps) {
+  const { variant = 'outline', children, error, disabled, style, ...rest } = props
+
+  const { theme, css } = useCss()
+
+  const baseStyles = createBaseStyles(theme)
+  const variantStyles = createVariantStyles(theme)
```

Comment:
```suggestion
  const styles = createStyles(theme, variant, disabled, invalid, clickable)
```
I think it's clearer. You can use `css(style1, style2,...)` inside this method

Reply:
done ✔️